client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -83,14 +135,17 @@ tie.factory('PythonCodePreprocessorService', [
             return START_INDENT + line;
           }
         });
-
-        var newCodeLines = [firstLine].concat(subsequentLines);
+        var newCodeLines = [firstLine].concat(systemFunctions);
+        newCodeLines = newCodeLines.concat(subsequentLines);
         return newCodeLines.join('\n');
       },
-      preprocessCode: function(code, mainFunctionName, correctnessTests) {
+      
+      preprocessCode: function(
+      code, mainFunctionName, correctnessTests, performanceTests) {
```

Comment:
Indent by 4 -- would usually be 2, but an additional 2 helps separate this line visually from the next.